### ISSUE TITLE
babel pre pro/postcessing checklist

### ISSUE THREAD

**Issue by erwinmombay** [ampproject/amphtml#18230]
- [x] remove assert code from java
  - [x] transform removal needs to be a post process since types are affected by these
- [ ] error transformation

**Comment by ampprojectbot** [comment]
This issue seems to be in  Pending Triage for awhile. @erwinmombay Please triage this to an appropriate milestone.

**Comment by rsimha** [comment]
@erwinmombay What's left to do here?

**Comment by rsimha** [comment]
Closing this in favor of #23960, which tracks the last remaining steps before we can remove `AmpPass.java` (#24047).
